examples/tensorflow/image_recognition/keras_models/resnet50/quantization/qat/main.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import codeop
+import logging
+import numpy as np
+import tensorflow as tf
+
+tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
+
+flags = tf.compat.v1.flags
+FLAGS = flags.FLAGS
+logger = logging.getLogger(__name__)
+
+## Required parameters
+flags.DEFINE_string(
+    'input_model', None, 'Run inference with specified keras model.')
+
+flags.DEFINE_string(
+    'output_model', None, 'The output quantized model.')
+
+flags.DEFINE_string(
+    'mode', 'performance', 'define benchmark mode for accuracy or performance')
+
+flags.DEFINE_bool(
+    'tune', False, 'whether to tune the model')
+
+flags.DEFINE_bool(
+    'benchmark', False, 'whether to benchmark the model')
+
+flags.DEFINE_string(
+    'dataset_location', None, 'location of the dataset on tfrecord format')
+
+flags.DEFINE_integer(
+    'batch_size', 32, 'batch_size')
+
+from neural_compressor.experimental.metric.metric import TensorflowTopK
```

Comment:
experimental all needs to be updated after https://github.com/intel/neural-compressor/pull/244 merged.

Reply:
They will be modified if PR244 can be merged before code freeze.